VALUES.md

```diff
@@ -0,0 +1,66 @@
+package management, libuv, V8 and Chakra, builds/ops, testing and its
+infrastructure, hardware and IoT, Android, streams, community events,
+facilitation, moderation and education. These and more are all concerns that
+can be handled separately and by members who might be much better qualified
+for these individual tasks than a central decision body.
+
+Ayo.js wants to go further than Node.js did in spreading responsibility across
+the project, so everyone can focus and specialize better than before.
+Contributors should be able to organize and manage their own subprojects
+in a way that is effective and efficient for their unique needs.
+
+## Humans are more important than red tape.
+
+If governance policy is not serving the values set forth here, that governance
+policy should change with the required agility. Following the governance policy
+to the letter should not take precedence over staying true to our stated values,
+not to mention the well-being of our humans.
+
+## Humans are more important than APIs and Benchmarks.
+
+Technical discussions are difficult, especially in large communities with
+members with all sorts of perspectives. You are bound to find yourself in
+situations with no clear consensus, with new concerns being brought up
+on a regular basis, and things that might seem insignificant in other
+contexts suddenly exploding.
+
+The Ayo.js project's perspective is that when this happens, it often takes a
+toll on the well-being of the members participating, or it prevents potential
+members from contributing again, if at all. It also recognizes that the
+difference between individual choices is less important than the ability to
+come to a perfect technical decision.
+
+Projects change, evolve, and find themselves dealing with new concerns all the
+time. In all technical matters, Ayo.js will take into account that in the end,
+it is the people making those decisions that matter most, not the decisions
+themselves. It’s ok for code to be imperfect.
+
+## Humans are more important than corporations.
+
+All sorts of corporations and other organizations around the world use and
+benefit from the software underlying Ayo.js. Many contributors, in fact, might
+have their efforts in the project directly funded by them. But this can often
+turn into such direct reliance on corporations that they can essentially seize
+control of a project — or struggle for power over it while individual
+contributors are left wondering if they can ever influence the project
+themselves.
+
+Ayo.js will explore new structures and framings for corporate relationships
+in such a way that its individual humans, not the corporations that employ them,
+are ultimately still the ones determining the direction of the project.
+Ayo.js should be a space where independent contributors, freelancers, hobbyists,
+and students all feel like they hold the same footing as corporate employees
+at all levels of the project.
```

Comment:
Do y'all think it'd be worth adding another heading expanding on our desire to create a welcoming space for aspiring contributors? The other values work to that end, but I believe it's worth strongly stating as its own explicit goal past the mention beyond the mention here and in the heading.

Reply:
Happy to take suggestions for wording if you have any :)

Reply:
I'll think on it! I'm about to head to bed but I'll mull it over and if nobody's beat me to it I'll take a crack at it tomorrow evening.

Reply:
After mulling it over, I think this is a natural extension of the other elements of the statements here. Others should feel free to add this if they think otherwise, but I'm happy with this where it is, and I'm feeling like I'd rather make an addition later than leave this in limbo longer than it needs to be.

Reply:
Newcomers may have a bit more trouble finding space than existing folks, so maybe it is worth it?

Or perhaps only in the other guidelines, unnecessary as an extra stated core value as this already covers that?